nncf/quantization/algorithms/hyperparameter_tuner/param_grid.py

```diff
@@ -70,5 +74,38 @@ def get_quantization_param_grid() -> Dict[str, Any]:
             for min_v, max_v in itertools.product(min_param_values, max_param_values)
         ],
     }
-
     return param_grid
+
+
+def _get_smooth_quant_param_grid() -> ParamGrid:
+    return {"advanced_parameters:smooth_quant_alpha": [0.15, 0.25, 0.5, 0.75, 0.95]}
+
+
+def _get_channel_alignment_param_grid() -> ParamGrid:
+    return {}
```

Comment:
Why is it empty since we add `ChannelAlignment` in this PR to the Tuner?

Reply:
The `ChannelAlignment` algorithm does not have any parameters to tune, but it is still applied to the model as a pipeline step.

Reply:
I thought that we should tune `ChannelAlignment` as the algorithm itself: apply it or not. Otherwise, I don't get why should we have a grid for this algo.
@alexsu52

Reply:
I have just one idea for how to handle this. I propose always including the `ChannelAlignment` algorithm in the Pipeline and adding an additional parameter, `disable_algorithm`, to the `ChannelAlignment.__init__()` method. This parameter would allow us to disable this algorithm when needed. What do you think?

Reply:
Looks like it's the same parameter with the `disable_channel_alignment`, but the new one keeps the `ChannelAlignment` instance alive. Since we had a problem with this behaviour, but with the `SmoothQuant` (see #2090), I propose to keep this case in mind.

Reply:
I've filed the ticket (121767) to support this feature.